src/web/tools/GestureHandlerWebDelegate.ts

```diff
@@ -119,6 +131,51 @@ export class GestureHandlerWebDelegate
     e.stopPropagation();
   }
 
+  private setUserSelect(isHandlerEnabled: boolean) {
+    const { userSelect } = this.gestureHandler.getConfig();
+
+    this.view.style['userSelect'] = isHandlerEnabled
+      ? userSelect ?? 'none'
+      : this.defaultViewStyles.userSelect;
+
+    this.view.style['webkitUserSelect'] = isHandlerEnabled
+      ? userSelect ?? 'none'
+      : this.defaultViewStyles.userSelect;
```

Comment:
I wonder how much we want to push restoring the previous value.

We could save the value here if a handler is enabled and restore it when it's disabled, though it still wouldn't be perfect (I don't think we can do perfect here). WDYT?

Reply:
So that would be situation where someone changes styles during gesture? I'm not exactly sure what are you referring to 😅

Reply:
Okay, we will keep current version as discussed. I'll change that if it starts to cause any problems.